### PR TITLE
Drop EL7 for ext db preparation steps

### PR DESCRIPTION
**Pull request by upadhyeammit**
Cherry-pick into:
none

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Thanks for the ping. Looks good to me as long as it's not cherry picked for 3.1.

**Comment by upadhyeammit** [comment]
> Thanks for the ping. Looks good to me as long as it's not cherry picked for 3.1.

Don't merge this, i think the change is incorrect. Pushing new changes

**Comment by upadhyeammit** [comment]
I misread the if statements, now from my side it looks correct. @maximiliankolb request to review.

File: guides/common/attributes-base.adoc [hunk 1]
```diff
@@ -29,6 +29,9 @@
 // Do not update the puppet4 repo versions. They must stay at 6.3.
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
 :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
+:RepoRHEL8ServerBaseos: rhel-8-for-x86_64-baseos-rpms
```
Comment (ekohl) [inline guides/common/attributes-base.adoc:32]: There is no RHEL 8 Server (nor a RHEL 8 Workstation), there is just RHEL 8. The name should reflect that. In https://github.com/theforeman/foreman-documentation/pull/1054 there was some work to clean it up. That's probably a much better base to work on.

**Review by maximiliankolb — APPROVED** [review]
This change is okay for me for now. I will create a PR once our downstream product is running on EL8.

**Comment by maximiliankolb** [comment]
> > Thanks for the ping. Looks good to me as long as it's not cherry picked for 3.1.
> 
> 
> 
> Don't merge this, i think the change is incorrect. Pushing new changes

Sorry I didn't notice. I had limited capabilities using the GitHub app.

File: guides/common/attributes-base.adoc [hunk 2]
```diff
@@ -29,6 +29,9 @@
...
+:RepoRHEL8ServerAppstream: rhel-8-for-x86_64-appstream-rpms
+:RepoRHEL8ServerSatelliteServerProductVersion: satellite-6.11-for-rhel-8-x86_64-rpms
```
Comment (ekohl) [inline guides/common/attributes-base.adoc:34]: While the others don't use a version macro either, they should. This prevents the inconsistency there is now (note 6.10 vs 6.11 here). In https://github.com/theforeman/foreman-documentation/pull/1054 we addressed that as well.

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by upadhyeammit** [comment]
Closing this pr as milestone to do these changes have been changed to Foreman 3.4

**Pull request closed without merging**